src/DIRAC/Core/Utilities/Graph.py

```diff
@@ -74,405 +68,4 @@ def _getProperty(self, name):
         classdict["makeProperty"] = makeProperty
         classdict["_setProperty"] = _setProperty
         classdict["_getProperty"] = _getProperty
-        return type.__new__(cls, name, bases, classdict)
-
-
-@six.add_metaclass(DynamicProps)
```

Comment:
The only left class here is DynamicProps which in turn is only used in OperationHandlerBase.py. It is better to move this class to the OperationHandlerBase.py and scrap Utilities/Graph.py altogether